explainer-server/app/controllers/ApiController.scala

```diff
@@ -32,9 +32,23 @@ class ApiController @Inject() (val publicSettingsService: PublicSettingsService)
     })
   }
 
-  override def update(id: String, fieldName: String, value: String): Future[Explainer] =
+  override def update(id: String, fieldName: String, value: String): Future[Explainer] = {
     ExplainerStore.update(id, Symbol(fieldName), value)
+  }
 
   override def load(id: String): Future[Explainer] = ExplainerStore.load(id)
 
+  override def create(): Future[Explainer] = ExplainerStore.create()
+
+  override def publish(id: String) = {
+    for {
+      explainer <- load(id)
+      _ <- update(id, "headline_published", explainer.headline)
+      _ <- update(id, "body_published"    , explainer.body)
```

Comment:
Um... that's a good question. What we discussed on Friday had two parts. Draft state versus Publish state and the history of changes. For the latter @annebyrne indicated that there was no immediate need for the feature and could actually confuse the users (that being said it can always be added later). The problem now becomes how to distinguish between draft versus published, and having the Explainer object become as below was the simplest while neat solution.
```
case class Explainer(
              id                     : String,
              headline               : String,
              body                   : String,
              last_update_time_milli : Option[Long],
              headline_published     : Option[String],
              body_published         : Option[String])
``

In particular, this implies that the generation of the JSON file that is currently sent to Front will simply present `headline_published` and `body_published` as data (fields renamed to fit the current file grammar, of course).